### ISSUE TITLE
Open .ipynb file from OS file manager

### ISSUE THREAD

**Issue by robertritz** [jupyterlab/jupyterlab-desktop#262]
### Problem

* Opening notebooks from a file manager (Finder on macOS, File Explorer on Windws) does not open the notebook in the JupyterLab standalone app.
* I would like it if double-clicking on a .ipynb file in a file manager opened that notebook in JupyterLab.
* New users especially don't always understand that you can't just open the file out of the box, and instead you need to open up an app first and then open that file. 

### Proposed Solution

* Opening a notebook from a file manager should open the notebook in JupyterLab
* The built-in file browser could also move to that location, but I believe since you are already in a file manager that would be unnecessary.

### Additional context

* Basically every desktop app today works in this way. I know it's early days still for the standalone JupyterLab app, but this would be a big thing for my students (and me).

**Comment by connorferster** [comment]
I agree, I think this is one of the big perks of having a desktop app. Would be amazing for `.ipynb` files to be "double clickable".

**Comment by connorferster** [comment]
I just checked my installation on Windows 10. I do have "double clickable" `.ipynb` files! It's great!

**Comment by oliviermarti** [comment]
Double clicking on a `.ipynb` now opens correctly the file in JupyterLab on my Mac, so I guess this issues could be close.

Anyway I have a request : when doing so with the last version (3.6.1-1), it opens a new session instead of opening a new tab in the existing session. An there is now way to move a tab between sessions. That's mean that my screen is rapidly cluttered by a lot of windows. 

Opening in the current session will be nice for me.

**Comment by mbektas** [comment]
@oliviermarti we will add support for reusing existing sessions when possible. closing this issue now since double click open is already available.
